melodic/distribution.yaml

```diff
@@ -2664,6 +2664,13 @@ repositories:
       url: https://github.com/HebiRobotics/hebi_cpp_api_ros.git
       version: master
     status: developed
+  hebi_description:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/HebiRobotics/hebi_description-release.git
+      version: 0.1.0-1
+    status: developed    
```

Comment:
This is failing CI because there is trailing whitespace after "developed".  Please fix and we can proceed.